### PR TITLE
Activate by default the first declared profile

### PR DESCRIPTION
**Pull request by lordofthejars**

### CODE REVIEW

**Comment by dipak-pawar** [comment]
@lordofthejars Can we do the same for container if it's not supported by chameleon?

**Comment by lordofthejars** [comment]
It is done in both cases.

**Comment by dipak-pawar** [comment]
ohh I missed that part.

**Comment by dipak-pawar** [comment]
LGTM

File: src/main/java/org/jboss/forge/arquillian/container/ProfileManager.java [hunk 1]
```diff
@@ -36,26 +37,31 @@
         return profiles;
     }
 
-    public void addProfile(Project project, Container container, List<Dependency> dependencies) {
+    public boolean isAnyProfileRegistered(Project project) {
+        MavenFacet mavenCoreFacet = project.getFacet(MavenFacet.class);
+        return mavenCoreFacet.getModel().getProfiles().size() != 0;
```
Comment (bartoszmajsak) [inline src/main/java/org/jboss/forge/arquillian/container/ProfileManager.java]: `!isEmpty` for my taste, but that's up to you ;)

**Review by bartoszmajsak — APPROVED** [review]
Just one thing :)

File: src/main/java/org/jboss/forge/arquillian/container/ProfileManager.java (hunk 1 above)
Comment (lordofthejars) [reply src/main/java/org/jboss/forge/arquillian/container/ProfileManager.java]: I always forget about this method.

**Pull request merged**